bucket/super-productivity.json

```diff
@@ -1,30 +1,39 @@
 {
-  "version": "6.1.2",
-  "description": "To-do list & time tracker for programmers and other digital workers with Jira, Github, and Gitlab integration.",
-  "homepage": "https://super-productivity.com/",
-  "license": "MIT",
-  "url": "https://github.com/johannesjo/super-productivity/releases/download/v6.1.2/superProductivity-Setup-6.1.2.exe#/dl.7z",
-  "hash": "bc7523cffb1e9d64ae7ab0760c7c4432b743cfa5503c386de2a01f7da554c664",
-  "installer": {
-    "script": [
-        "Expand-7zipArchive \"$dir\\`$PLUGINSDIR\\app-64.7z\" \"$dir\"",
-        "Remove-Item \"$dir\\`$PLUGINSDIR\" -Force -Recurse"
-    ]
-  },
-  "uninstaller": {
-    "file": "Uninstall superProductivity.exe"
-  },
-  "bin": "superProductivity.exe",
-  "shortcuts": [
-      [
-          "superProductivity.exe",
-          "superProductivity"
-      ]
-  ],
-  "checkver": {
-      "github": "https://github.com/johannesjo/super-productivity/releases"
-  },
-  "autoupdate": {
-      "url": "https://github.com/johannesjo/super-productivity/releases/download/v$version/superProductivity-Setup-$version.exe#/dl.7z"
-  }
+    "version": "6.1.2",
+    "description": "To-do list & time tracker for programmers and other digital workers with Jira, Github, and Gitlab integration.",
+    "homepage": "https://super-productivity.com/",
+    "license": "MIT",
+    "url": "https://github.com/johannesjo/super-productivity/releases/download/v6.1.2/superProductivity-Setup-6.1.2.exe#/dl.7z",
+    "hash": "bc7523cffb1e9d64ae7ab0760c7c4432b743cfa5503c386de2a01f7da554c664",
+    "architecture": {
+        "64bit": {
+            "installer": {
+                "script": [
+                    "Expand-7zipArchive \"$dir\\`$PLUGINSDIR\\app-64.7z\" \"$dir\"",
+                    "Remove-Item \"$dir\\`$PLUGINSDIR\" -Force -Recurse"
+                ]
+            }
+        }
+    },
+    "uninstaller": {
+        "file": "Uninstall superProductivity.exe"
+    },
```

Comment:
```suggestion
```

Reply:
Should I remove the uninstaller property?

Reply:
Removing the uninstaller property does not bring any issues.
Will be committed.